src/operator/tensor/ordering_op-inl.h

```diff
@@ -190,8 +191,8 @@ using namespace mshadow;
 
 
 struct fill_ind_to_one {
-  template<typename DType>
-  MSHADOW_XINLINE static void Map(int i, const index_t* indices, DType* out) {
```

Comment:
data type of `i` is `int`. Would this actually cause out of bound error with large inputs?

Reply:
makes sense will update the PR